code/game/gamemodes/vampire/vampire_powers.dm

```diff
@@ -0,0 +1,347 @@
+/obj/effect/proc_holder/spell
+	var/gain_desc
+	var/blood_used = 0
+	var/vamp_req = FALSE
+
+/datum/vampire_passive
+	var/gain_desc
+
+/datum/vampire_passive/New()
+	..()
+	if(!gain_desc)
+		gain_desc = "You have gained \the [src] ability."
+
+
+///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+/datum/vampire_passive/regen
+	gain_desc = "Your rejuvination abilities have improved and will now heal you over time when used."
+
+/datum/vampire_passive/vision
+	gain_desc = "Your vampiric vision has improved."
+
+/datum/vampire_passive/full
+	gain_desc = "You have reached your full potential and are no longer weak to the effects of anything holy and your vision has been improved greatly."
```

Comment:
"You have reached your full potential: you are no longer weak to the effects of anything holy and your vision has been improved greatly."